tests/Task.test.ts

```diff
@@ -1150,6 +1175,83 @@ describe('toggle done', () => {
     });
 });
 
+describe('handle new status', () => {
+    beforeAll(() => {
+        sampleStatusesForToggling.forEach((s) => {
+            StatusRegistry.getInstance().add(Status.createFromImportedValue(s));
+        });
+    });
+
+    afterAll(() => {
+        StatusRegistry.getInstance().resetToDefaultStatuses();
+    });
+
+    beforeEach(() => {
+        jest.useFakeTimers();
+        jest.setSystemTime(new Date('2023-06-26'));
+    });
```

Comment:
This should have had an `afterEach` - I'll be adding one later at the top of the file, so it's not worth dealing with here.